python_modules/dagster/dagster/_core/definitions/asset_layer.py

```diff
@@ -297,9 +297,6 @@ def asset_key_to_dep_node_handles(
             {}
         )  # memoized map of node output handles to all node output handle dependencies that are from ops
         for output_name, asset_key in assets_defs.keys_by_output_name.items():
-            output_def = assets_defs.node_def.output_def_named(output_name)
-            output_name = output_def.name
-
```

Comment:
Not sure exactly why this code is here as `output_name` appears redundant with `output_name` set in the loop. cc @clairelin135

Reply:
yeah this makes sense to me! Agree that these lines are redundant, as long as bk tests still pass this should be fine 👍

Reply:
Cool-- actually mind reviewing this whole PR? It's the last one in the stack, alex did all the others